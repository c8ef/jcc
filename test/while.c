int main(void) {
  while (1) {
    int x = 42;
  }
}
