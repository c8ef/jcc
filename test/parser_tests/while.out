FunctionDecl: main
  Args[0]
  CompoundStatement
    WhileStatement
      IntergerLiteral: 1
      CompoundStatement
        DeclStatement
          VarDecl: x
            IntergerLiteral: 42
